gitlens/gitlens-release-notes-current.md

```diff
@@ -714,7 +714,7 @@ Please note that while using cloud workspaces requires a free account, shared cl
 
 Workspaces can be accessed in the new GitKraken Workspaces view in the GitLens side bar.
 
-## ✨ Commit Graph Enhancements
+## `PRO` Commit Graph Enhancements
```

Comment:
On all the headers before this you have `PRO` added at the end. Then from here onward it's added at the beginning of the header. I would make them all consistent.